CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 
 - Fixed a bug when `Dataframe.join_table_function` did not run all the necessary queries to setup join table function when sql simplifier was enabled.
 - Added comments to SnowflakeFile and added a API reference page for it.
+- Fixed a bug where `DataFrameWriter.save_as_table` and `DataFrame.copy_into_table` fail to parse fully qualified table name.
```

Comment:
fail -> failed
table name -> table names

Reply:
@sfc-gh-aling I'll take care of these comments when I merge with release branch. If you push another commit fixing this, we'll rerun tests. I'm trying to avoid that